### PR TITLE
Swagger & Kafka group ID fixes

### PR DESCRIPTION
**Pull request by mmacai**
- fixed missing id field for message swagger spec
- fixed random generation of Kafka group id

Test:
- `mix test` & integration tests should pass
- run rig locally - sending `/message` form `http://localhost:4010/swagger-ui` should work on ootb
- same should work from docker container

- run rig with Kafka => check consumer => all should have consumer group set to `rig` - can be changed via env var

### CODE REVIEW

File: apps/rig_api/lib/rig_api/controllers/message_controller.ex [hunk 1]
```diff
@@ -60,6 +60,14 @@ defmodule RigApi.MessageController do
...
 
           properties do
+            id(
+              :string,
+              "ID of the event. The semantics of this string are explicitly undefined to ease \
+              the implementation of producers. Enables deduplication.",
+              required: true,
+              example: "A database commit ID"
+            )
+
```
Comment (kevinbader) [inline apps/rig_api/lib/rig_api/controllers/message_controller.ex]: isn't `source` required too?

File: apps/rig_kafka/lib/rig_kafka/config.ex [hunk 2]
```diff
@@ -51,7 +51,7 @@ defmodule RigKafka.Config do
       consumer_topics: Map.get(config, :consumer_topics, []),
       server_id: Map.get(config, :server_id) || String.to_atom("rig_kafka_#{uuid}"),
       client_id: Map.get(config, :client_id) || String.to_atom("brod_client_#{uuid}"),
-      group_id: Map.get(config, :group_id) || "brod_group_#{uuid}",
+      group_id: Map.get(config, :group_id) || "brod_group_rig",
```
Comment (kevinbader) [inline apps/rig_kafka/lib/rig_kafka/config.ex]: Why is the default here different from the default in `config.exs`? Actually, I'm not sure a default makes sense here in the first place - after all, we already define defaults in the config files.

File: website/i18n/en.json [hunk 3]
```diff
@@ -37,10 +37,6 @@
         "title": "Metrics Details",
         "sidebar_label": "Metrics Details"
       },
-      "https": {
-        "title": "HTTPS",
-        "sidebar_label": "HTTPS"
-      },
```
Comment (kevinbader) [inline website/i18n/en.json]: not sure why this is here

File: integration_tests/kafka_tests/run.sh [hunk 4]
```diff
@@ -22,7 +22,7 @@ while ! is_kafka_ready; do
     sleep 1
 done
 
-check whether Kafka is now ready or the loop has been aborted:
+# check whether Kafka is now ready or the loop has been aborted:
```
Comment (kevinbader) [inline integration_tests/kafka_tests/run.sh]: we don't run them often, do we? :D

File: apps/rig_api/lib/rig_api/controllers/message_controller.ex (hunk 1 above)
Comment (mmacai) [reply apps/rig_api/lib/rig_api/controllers/message_controller.ex]: Should be, but apparently not in our code. At least I didn't see any error when using `/messages` endpoint.

File: apps/rig_kafka/lib/rig_kafka/config.ex (hunk 2 above)
Comment (mmacai) [reply apps/rig_kafka/lib/rig_kafka/config.ex]: Left it there due to tests, I'll remove it and put in test config.

File: integration_tests/kafka_tests/run.sh (hunk 4 above)
Comment (mmacai) [reply integration_tests/kafka_tests/run.sh]: Actually I made some improvements in Avro PR -- so should be better in near future :D.

File: apps/rig_api/lib/rig_api/controllers/message_controller.ex (hunk 1 above)
Comment (kevinbader) [reply apps/rig_api/lib/rig_api/controllers/message_controller.ex]: created an issue for this: #191

**Pull request merged**